src/frontends/lean/json.cpp

```diff
@@ -4,8 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 
 Author: Gabriel Ebner
 */
-#if defined(LEAN_SERVER)
```

Comment:
Oops, I didn't remember the flag was introduced for the JSON library. I guess I'll need a few more `#if`s in `info_manager` then, urgh.